server/app/lib/actions/fusor/fusor_base_action.rb

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+module Actions
+  module Fusor
+    class FusorBaseAction < Actions::Base
+      def plan(deployment=nil)
+        # Create deployment specific log file.
+        ::Fusor.log_change_deployment(deployment)
+      end
```

Comment:
I would probably roll some middleware instead:

```ruby
module Actions
  module MiddleWare
    class DeploymentLogger < Dynflow::Middleware
      def plan_phase(*args)
        # Create deployment specific log file.
        ::Fusor.log_change_deployment(action.input['deployment'])
      end
   end
end
```

Then for FusorBaseAction:

```ruby
module Actions
  module Fusor
    class FusorBaseAction < Actions::Base
      include ::Actions::Deployment::DeploymentLogger
    end
  end
end
```

This will save you from calling `super` everywhere.